receiver/podmanreceiver/factory.go

```diff
@@ -34,7 +34,6 @@ func createDefaultConfig() *Config {
 	return &Config{
 		ScraperControllerSettings: cfg,
 		Endpoint:                  "unix:///run/podman/podman.sock",
-		Timeout:                   5 * time.Second,
```

Comment:
Is 5s no longer default? If so need to update https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/podmanreceiver/README.md?plain=1#L35

Reply:
The timeout is set to 5 seconds on like 32 (I missed it at first)

Reply:
if you look at line 32 above, it's still the default

Reply:
ah that's silly. OK then.